src/main/resources/locales/en-US.yml

```diff
@@ -24,7 +24,7 @@ general:
     player-has-no-island: "&cThat player has no island!"
     already-have-island: "&cYou already have an island!"
     no-safe-location-found: "&cCould not find a safe spot to teleport you to on the island."
-    not-owner: "&cYou are not the owner of your island!"
+    not-owner: "&cYou are not the owner of the island!"
```

Comment:
Unsure about this one, but I guess it makes more sense with this revision.

Reply:
`not-ower` is used in contexts for commands which people can only execute on their island, so if player A executes a command they have no permission to on the island of player B it returns that message, yet it doesn't make much sense in its current state.

Reply:
Okay, understold. Great!